File: Lesson09_OOP/06_NotOO/CalculatorNotOO.py
"""
Copyright Khawla Shnaikat and ChatGPT 3.5, 2023
Licensed under GPL v3
See LICENSE.txt for more information.
"""
def add(one, two):
    return one + two

def subtract(one, two):
    return one - two

def multiply(one, two):
    return one * two

def divide(one, two):
    if two == 0:
        return 0
    return one / two

def main():
    one = 9.0
    two = 0.1
    print("Add:", add(one, two))
    print("Subtract:", subtract(one, two))
    print("Multiply:", multiply(one, two))
    print("Divide:", divide(one, two))

if __name__ == "__main__":
    main()

File: Lesson09_OOP/06_NotOO/CalculatorNotOO.java
/*
Copyright Ann Barcomb and Emily Marasco, 2022
Licensed under GPL v3
See LICENSE.txt for more information.
*/

public class CalculatorNotOO {

    public static double add(double one, double two) { return one + two; }
    public static double subtract(double one, double two) { return one - two; }
    public static double multiply(double one, double two) { return one * two; }
    public static double divide(double one, double two) { 
        if (two == 0) { return 0; }
        return one/two;
    }

    public static void main(String[] args) {
        double one = 9.0;
        double two = 0.1;
        System.out.println("Add: " + add(one, two));
        System.out.println("Subtract: " + subtract(one, two));
        System.out.println("Multiply: " + multiply(one, two));
        System.out.println("Divide: " + divide(one, two));
    }
}


